Formula/openssh.rb

```diff
@@ -53,9 +57,46 @@ def install
     # potential to break scripts, so recreate it for now.
     # Debian have done the same thing.
     bin.install_symlink bin/"ssh" => "slogin"
+
+    # https://opensource.apple.com/source/OpenSSH/OpenSSH-209.30.4/com.openssh.sshd.sb
```

Comment:
It can be done either way. If you prefer a resource I can do that.

My hesitation was that Apple's `opensource` website used to have some unreliability around SSL/TLS connections, particularly in the _"remember to update your cert before it expires"_ department, but to be fair that situation seems to have improved.